x-pack/plugins/ml/public/shared_components/single_metric_viewer/single_metric_viewer.tsx

```diff
@@ -150,11 +153,11 @@ const SingleMetricViewerWrapper: FC<SingleMetricViewerPropsWithDeps> = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const resizeHandler = useCallback(
     throttle((e: { width: number; height: number }) => {
-      if (Math.abs(chartWidth - e.width) > minElemAndChartDiff) {
-        setChartWidth(e.width);
+      if (Math.abs(chartDimensions.width - e.width) > minElemAndChartDiff) {
```

Comment:
I suspect this needs to be updated to check the height too, this is what might be causing the chart height not being propagated when you change the height of a panel only.

Reply:
Yep, good spot - updated in a5474353d89360f539a1f68abd0ded0be695f08f